client/components/application/RenderCHFSElement.tsx

```diff
@@ -1,117 +1,9 @@
-import Image from 'next/image';
 import { API_ENDPOINT } from '../../constants';
 import { useHttp } from '../../services/useHttp';
-import { Button } from '../generic';
-
-const TYPE_AS_STRING = [
-  'textfield',
-  'simpletextfield',
-  'textarea',
-  'simpletextarea',
-  'radio',
-  'simpleradios',
-  'select',
-  'currency',
-  'day',
-  'phoneNumber',
-];
+import { NOT_TO_BE_RENDERED, renderElement } from 'helpers';
 
 const NESTED_COMPONENTS = ['simplecols2'];
 
-const NOT_TO_BE_RENDERED = ['button', 'simplecontent', 'htmlelement'];
```

Comment:
I think that's what is being used to filter layout components. Can you use it to remove layout `columns` for example?